package.json

```diff
@@ -370,20 +370,28 @@
               ]
             }
           },
+          {
+            "label": "Go: Attach to local process",
+            "description": "Attach to an existing process by process ID",
+            "body": {
+              "name": "${1:Attach to Process}",
+              "type": "go",
+              "request": "attach",
+              "mode": "local",
+              "processId": 0
+            }
+          },
           {
             "label": "Go: Connect to server",
             "description": "Connect to a remote headless debug server",
             "body": {
-              "name": "${3:Connect to server}",
+              "name": "${1:Connect to server}",
               "type": "go",
-              "request": "launch",
+              "request": "attach",
               "mode": "remote",
-              "remotePath": "^\"\\${workspaceFolder}${1:}\"",
+              "remotePath": "^\"\\${workspaceFolder}\"",
               "port": 2345,
-              "host": "127.0.0.1",
-              "program": "^\"\\${workspaceFolder}${1:}\"",
-              "env": {},
-              "args": []
+              "host": "127.0.0.1"
```

Comment:
This wouldn't work without a process id right? If so, then we should add `processId` into this snippet

Reply:
This is connecting to an existing dlv server so no process ID is required.